stargazer/stargazer.py

```diff
@@ -530,6 +544,27 @@ def generate_additional_notes(self):
 class LaTeXRenderer(Renderer):
     fmt = 'LaTeX'
 
+    # LaTeX escape characters, borrowed from pandas.io.formats.latex
+    _ESCAPE_CHARS = [
+        ('\\', r'\textbackslash '),
+        ('_', r'\_'),
+        ('%', r'\%'),
+        ('$', r'\$'),
+        ('#', r'\#'),
+        ('{', r'\{'),
+        ('}', r'\}'),
+        ('~', r'\textasciitilde '),
+        ('^', r'\textasciicircum '),
+        ('&', r'\&')
+    ]
+
+    def _escape(self, text):
+        """Escape LaTeX special characters"""
+        if 'escape' in self.kwargs and self.kwargs['escape']:
```

Comment:
Cleaner: ``if self.kwargs.get('escape', False):``

Reply:
Cleaner indeed! Thanks for spotting this.